packages/types/index.ts

```diff
@@ -8,13 +8,14 @@ export interface idOSHuman {
 export interface idOSCredential {
   id: string;
   human_id: string;
-  issuer: string;
-  original_id: string;
-  credential_type: string;
-  credential_level: string;
-  credential_status: idOSCredentialStatus;
+  issuer_auth_public_key: string;
+  original_id?: string;
+  public_notes: string;
+  public_notes_signature: string;
+  broader_signature: string;
```

Comment:
The signatures don't live in the table anymore. They're passed in as arguments, checked against the expected values, and forgotten.